apps/playground/src/main.css

```diff
@@ -0,0 +1,9 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+html {
+  font-family: system-ui, sans-serif;
```

Comment:
shouldnt we take this from css package?

Reply:
I don't want to add webfont right now (not the point of the PR) but we could do it in a future PR

Reply:
I dont mean adding webfont. I mean defining font family.
It is done with itwinui-css, so you should not need to define anything for demo app.